src/tests/tests.py

```diff
@@ -48,6 +48,16 @@ def test_response_invalid_username(self):
         )
         self.assertEqual(response.status_code, 403)
 
+    def test_response_invalid_username_missing(self):
+        setup_test_environment()
+        client = Client()
+        response = client.get(
```

Comment:
We can (and should!) though probably better to make that change in a separate issue/PR.